modules/client/src/main/scala/lucuma/itc/client/InstrumentModesInput.scala

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc.client
+
+import cats.syntax.functor.*
+import io.circe.Encoder
+import io.circe.Json
+import io.circe.syntax.*
+import lucuma.core.enums.GmosNorthFilter
+import lucuma.core.enums.GmosNorthFpu
+import lucuma.core.enums.GmosNorthGrating
+import lucuma.core.enums.GmosSouthFilter
+import lucuma.core.enums.GmosSouthFpu
+import lucuma.core.enums.GmosSouthGrating
+import lucuma.itc.client.json.syntax.*
+
+sealed trait InstrumentModesInput
+
+object InstrumentModesInput {
+
+  final case class GmosNorth(
+    grating: GmosNorthGrating,
+    filter:  Option[GmosNorthFilter],
+    fpu:     GmosNorthFpu
```

Comment:
This bit differs from the GraphQL in that it requires a builtin.  I'll circle back and make it match the API.

Reply:
WDYM differs from the odb api?

Reply:
I mean, it differs from the ITC GraphQL API which supports `builtin` and `customMask`.  I'll change it to match the API.